wagtail/images/templates/wagtailimages/images/index.html

```diff
@@ -45,6 +48,23 @@
                         <input type="hidden" name="tag" value="{{ current_tag }}" />
                     {% endif %}
                 {% endif %}
+                <li>
+                    <div class="field choice_field select">
+                        <label for="order_images_by">{% trans "Sort by" %}:</label>
+                        <div class="field-content">
+                            <div class="input">
+                                <select id="order_images_by" name="ordering">
+                                    {% for ordering, ordering_text in ORDERING_OPTIONS.items %}
+                                        <option value="{{ ordering }}" {% if current_ordering == ordering %}selected="selected"{% endif %}>
+                                            {% trans ordering_text %}
```

Comment:
I did not notice this, but it is no longer required now that ordering_text is a `_()` wrapped value

Reply:
Okay, I'll need to look more into how i18n works. I've removed it.